### PR TITLE
C375258 Query search | Search by "Series" field of linked "MARC Bib" record (spitfire) (TaaS)

### PR DESCRIPTION
**Pull request by manowarqqq**
https://foliotest.testrail.io/index.php?/cases/view/375258
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5091/allure/#suites
image

### CODE REVIEW

**Review by ernestyedigaryan1 — APPROVED** [review]

**Review by zentestuken — CHANGES_REQUESTED** [review]
Also, please merge the latest master and verify test is passing with latest updates (see https://github.com/folio-org/stripes-testing/pull/2200). Some of your files may also require similar updates

File: cypress/e2e/inventory/search/search-by-series-field-of-linked-record.cy.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numberOfRecords: 1,
+    },
+  ],
+};
+
+describe('Inventory', () => {
+  describe('Search in Inventory', () => {
+    before('Create test data', () => {
+      cy.getAdminToken();
```
[the remark this answers is not in the document]
Comment (manowarqqq) [reply cypress/e2e/inventory/search/search-by-series-field-of-linked-record.cy.js:92]: done

File: cypress/e2e/inventory/search/search-by-series-field-of-linked-record.cy.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        InventoryInstances.resetAllFilters();
+      }
+      cy.createTempUser([Permissions.inventoryAll.gui]).then((userProperties) => {
+        testData.user = userProperties;
+      });
+      cy.logout();
+    });
+
+    after('Delete test data', () => {
+      Users.deleteViaApi(testData.user.userId);
```
[the remark this answers is not in the document]
Comment (manowarqqq) [reply cypress/e2e/inventory/search/search-by-series-field-of-linked-record.cy.js:159]: done

**Comment by manowarqqq** [comment]
> Also, please merge the latest master and verify test is passing with latest updates (see #2200). Some of your files may also require similar updates

https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5374/allure/#suites
image

**Review by zentestuken — APPROVED** [review]

**Pull request merged**